tests/asgi/test_ws.py

```diff
@@ -341,7 +341,9 @@ async def on_websocket(self, req, ws):
                             await asyncio.sleep(
                                 0
                             )  # Ensure recv_task() has a chance to get ahead
-                            await asyncio.wait([recv_task, ws.close(4099)])
+                            await asyncio.wait(
+                                [recv_task, asyncio.create_task(ws.close(4099))]
```

Comment:
this fixes the warning 
`The explicit passing of coroutine objects to asyncio.wait() is deprecated since Python 3.8, and scheduled for removal in Python 3.11.`

Reply:
this hangs in py36 -.-'

Reply:
fixed now